quick-start.md

```diff
@@ -7,55 +7,82 @@ description: Get an instance of TinyMCE up and running in less than 5 minutes.
 keywords: tinymce script textarea
 ---
 
-{{site.productname}} {{site.productmajorversion}} is a powerful and flexible rich text editor that can be embedded in the user's web application.
-
-{{site.productname}} {{site.productmajorversion}} is perfect for developers who want to see how the new version of {{site.productname}} integrates into their ecosystem.
+{{site.productname}} {{site.productmajorversion}} is a powerful and flexible rich text editor that can be embedded in web applications. This quick start covers how to add a {{site.productname}} editor to a web page using the {{site.cloudname}}.
 
 ## Step 1: Include the TinyMCE script
 
-Include this line of code in the `<head>` of the HTML page and link to the `tinymce.min.js` source file. Substitute 'no-api-key' with your api key.
+Include the following line of code in the `<head>` of a HTML page.
 
 ```html
 <script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
 ```
 
-> Tip: A complete HTML snippet is provided in Step 2.
+## Step 2: Initialize TinyMCE as part of a web form
 
+Initialize {{site.productname}} {{site.productmajorversion}} on any element (or elements) on the web page.
 
-## Step 2: Initialize TinyMCE as part of a web form
+{{site.productname}} enables identifying replaceable elements using a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors). Specify the element by passing an object that contains a `selector` to `tinymce.init()`, such as:
```

Comment:
This and the para above feels a bit awkward. The "identifying replaceable elements" and such seems a bit hard to parse. Could we combine them and simplify it a bit? E.g. 

`Initialize {{site.productname}} {{site.productmajorversion}} on any element (or elements) on the web page by passing an object that contains a `selector` value to `tinymce.init()`. The `selector` value can be any valid [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors).`

Reply:
fixed